### PR TITLE
Hotfix capture with empty UUID

### PR DESCRIPTION
**Pull request by evgeek**
Hotfix for https://github.com/PostHog/posthog-go/issues/69

### CODE REVIEW

File: posthog.go [hunk 1]
```diff
@@ -462,7 +462,7 @@ func (c *client) send(msgs []message) {
 
 // Upload serialized batch message.
 func (c *client) upload(b []byte) error {
-	url := c.Endpoint + "/batch/"
+	url := c.Endpoint + "/capture/"
```
Comment (dmarticus) [inline posthog.go:465]: wait, I don't understand; you're removing the `/batch` functionality?  I don't think that's a good long-term fix for this; folks are using that piece, too.  If we're looking to fix something quickly, we should just revert the change and then fix it more permanently instead of continuing to change more things.

Comment (evgeek) [reply posthog.go:465]: Yes, it is just a quick (not good) fix, i describe my thoughts here - https://github.com/PostHog/posthog-go/pull/68#issuecomment-2308798201.

Do you think you can configure `/batch` on the server side to work with an empty uuid, like `/capture`? If (I hope) this works out, previous PR will be ok. If not, we will need to think about how best to implement the functionality from the SDK side.

But for now you're right - rollback is better. I don't know all the SDK use cases and can't be sure it won't break something else.
Rollback PR - https://github.com/PostHog/posthog-go/pull/71
